_pages/Home/components/Login/index.js

```diff
@@ -8,17 +8,15 @@ const cookies = new Cookies();
 
 export function LoginForm() {
   const router = useRouter();
-  // if (cookies.get('token')) {
-  //   router.push('/dashboard');
-  // }
+  if (cookies.get('token')) {
+    router.push('/dashboard');
+  }
   return (
     <div className={styles.form}>
       <h1>Login</h1>
       <Formik
         initialValues={{ email: 'stephen-1310@hotmail.com', password: '' }}
         onSubmit={async (values) => {
-          // await new Promise((resolve) => setTimeout(resolve, 500));
```

Comment:
Not a biggy, but not good practice to have commented code without a reason.

In future it's good to put something like this before the commented code:

`// TODO: resolve promise after half a second`

Then it is obvious to any reviewer why the code has been commented out!